### ISSUE TITLE
Docker (OSX)

### ISSUE THREAD

**Issue by lasers** [PCSX2/pcsx2#424]
I wanted to know if somebody tried putting pcsx2 (stable/unstable) on a linux distro for Docker... to be used for OSX. Something like https://github.com/thewtex/docker-opengl-nvidia + PCSX2. I asked this question because OSX is terribly behind on updates. Is this even a fesaible option?

**Comment by uyjulian** [comment]
It's better to use Wine if you want to use later PCSX2 updates.

**Comment by lasers** [comment]
@uyjulian I went to ##winehq channel for support with pcsx2. I was there because I had an issue with winetricks not playing nicely with latest redist. I talked with a nice person there who was kind enough to provide me with a last-minute patch.

"MS updated the redist to build 12.0.30501 and winetricks needs to catch up again, hence the sha1sum mismatch".

I came back to update you with the latest information, I happened to check "brew info wine" and just happened to notice that you're the responsible for wine! :+1: 

Is it possible for you to come up with wine-beta (or wine-dev)?

The reason is that is (wine 1.6.2) "Although stable, it's considered rather outdated given the large number of bugs/features in 1.7.x (with 1.7.3.4 now latest)." regarding the issue where the pcsx2-installer.exe bundles vc++ 2013 installer and if I keep it selected (as component) (in wine 1.6.2), wine will run into an installer bug (named pupes).

One can deselect the runtime, then no winetricks is needed (because it will be already in wine-1.7.x) which pcsx2 developers might find this more favoring in term of OSX support. Thoughts?
